pkg/kubelet/kubelet.go

```diff
@@ -1626,18 +1626,15 @@ func (kl *Kubelet) syncPod(pod *api.Pod, mirrorPod *api.Pod, runningPod kubecont
 		return err
 	}
 
-	// Starting phase:
-	ref, err := api.GetReference(pod)
```

Comment:
Thanks for your riview.
Yes, if there are many use places, old logic is better.
But ref is only used here now.  So i think we should add check before use it because api.GetReference(pod) may get an err. So ref may be nil.